src/proxy-ui-api/frontend/src/locales/en.json

```diff
@@ -27,6 +27,10 @@
       "parameters": "Service Parameters"
     }
   },
+  "backup": {
+    "backupConfiguration": "Back up config.",
```

Comment:
I think that `Back up config.` is correct in this case since we want to express that by clicking this button you will create a backup of the current configuration. But if you meant the one below this, `Upload back up`, it was wrong and is now fixed to `Upload backup`